MauroDataMapper/mdm-ui
# Feature/gh 845 merge

Resolves #854
requires security plugin update https://github.com/MauroDataMapper/mdm-core/pull/460

Allows the merging of branches into main branches of datamodels and versionfolders, update include ui fixes, usability improvements and bugfixes.

to test:
Publish the security snapshot to maven local.
add it to your gradle build for mdm-application build
![image](https://github.com/MauroDataMapper/mdm-ui/assets/85611988/6dfadc91-c885-4c1a-acf8-cb336cc3583b)
Clean and build

Create the following, one branch, one main
![image](https://github.com/MauroDataMapper/mdm-ui/assets/85611988/bc0bf414-1578-4aad-9cb1-5337a2cedef3)

Select merge on the main branch
![image](https://github.com/MauroDataMapper/mdm-ui/assets/85611988/68bc9547-935c-4d9d-97b9-31276b1eeffb)
 
note that the branch is initially empty
![image](https://github.com/MauroDataMapper/mdm-ui/assets/85611988/05a33352-58d2-41f5-b6ab-0451cf5e3510)

select a mergeable branch
![image](https://github.com/MauroDataMapper/mdm-ui/assets/85611988/1c729077-a472-427a-895f-fa2058b5d5e1)
